src/style.css

```diff
@@ -0,0 +1,109 @@
+@font-face {
+  font-family: AvenirBook;
+  src: url(fonts/Avenir-Book.ttf) ;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+body {
+  margin-top: 0;
+}
+
+.wraper-big {
+  margin-bottom: 20px;
+  position: relative;
+  margin-top: 20px;
+}
+
+.wraper-small {
+  position: relative;
+}
+
+input[type="search"]::-webkit-search-cancel-button {
+  opacity: 0;
+}
+
+.icon-big {
+  color: #687c94;
+  position: absolute;
+  width: 19px;
+  bottom: 26px;
+  left: 26px;
+}
+
+.icon-small {
+  position: absolute;
+  color: #687c94;
+  width: 11px;
+  bottom: 16px;
+  left: 13px;
+}
+
+.big {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  font-family: AvenirBold, AvenirBook, sans-serif;
+  padding-left: 62px;
+  line-height: 22px;
+  font-size: 16px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.small {
+  height: 42px;
+  box-sizing: border-box;
+  width: 100%;
+  font-family: AvenirBold, sans-serif;
+  padding-left: 33px;
+  line-height: 19px;
+  font-size: 14px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big:focus-within {
```

Comment:
you don't have any descendants of this element so you can just use :focus pseudoclass